src/clout/core.cljc

```diff
@@ -0,0 +1,162 @@
+(ns clout.core
+  "A small language for routing."
+  (:require [clojure.string :as string]
+            [clojure.set :as set]
+            [instaparse.core :as insta]
+            ))
```

Comment:
Please make sure your code adheres to the [Clojure Style Guide](https://github.com/bbatsov/clojure-style-guide), so no hanging `))`.